src/controllers/salaryController.ts

```
@@ -57,10 +57,11 @@ export async function postSalaryDisburse(
     });
   } catch (e) {
     if (e instanceof z.ZodError) {
-      return next(new AppError(e.errors.map((x) => x.message).join("; "), 400));
+      throw new AppError("Validation error", 400, "VALIDATION_ERROR", e.flatten());
```

ESLint

[error] Replace `"Validation·error",·400,·"VALIDATION_ERROR",·e.flatten()` with `⏎········"Validation·error",⏎········400,⏎········"VALIDATION_ERROR",⏎········e.flatten(),⏎······`

(prettier/prettier)